sublime_music/dbus/manager.py

```diff
@@ -72,23 +72,24 @@ def dbus_name_acquired(connection: Gio.DBusConnection, name: str):
                 with open(spec_path) as f:
                     node_info = Gio.DBusNodeInfo.new_for_xml(f.read())
 
-                connection.register_object(
-                    "/org/mpris/MediaPlayer2",
-                    node_info.interfaces[0],
-                    self.on_method_call,
-                    self.on_get_property,
-                    self.on_set_property,
-                )
+                    connection.register_object(
+                        "/org/mpris/MediaPlayer2",
+                        node_info.interfaces[0],
+                        self.on_method_call,
+                        self.on_get_property,
+                        self.on_set_property,
+                    )
```

Comment:
was this change necessary?

Reply:
no, but node_info is defined at that level

Reply:
It's better to keep the `with` statements as short as possible. Only the `f.read()` needs to be within the `with`, so please unindent this. Since it's within the same scope, Python handles this just fine